FND/Organizations/FormalOrganizations.rdf

```diff
@@ -93,12 +173,77 @@
 		<skos:definition>organization that is not formally constituted in some way</skos:definition>
 	</owl:Class>
 	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-org-fm;employs">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;playsActiveRoleThatDirectlyAffects"/>
+		<rdfs:label>employs</rdfs:label>
+		<rdfs:range rdf:resource="&fibo-fnd-aap-ppl;Person"/>
+		<skos:definition>indicates someone that is employed by the legal person or organization</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-org-fm;hasEmployedParty">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-org-org;hasOrganizationMember"/>
+		<rdfs:label>has employed party</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-org-fm;Employment"/>
+		<rdfs:range rdf:resource="&fibo-fnd-org-fm;Employee"/>
+		<owl:inverseOf rdf:resource="&fibo-fnd-org-fm;isEmployedIn"/>
+		<skos:definition>identifies the employee in an employment situation</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-org-fm;hasEmployee">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;actsOn"/>
+		<rdfs:label>has employee</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-org-fm;Employer"/>
+		<rdfs:range rdf:resource="&fibo-fnd-org-fm;Employee"/>
+		<owl:inverseOf rdf:resource="&fibo-fnd-org-fm;isEmployeeOf"/>
+		<skos:definition>indicates an employee that is employed by the employer</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-org-fm;hasEmployingParty">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-org-org;hasMembership"/>
+		<rdfs:label>has employing party</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-org-fm;Employment"/>
+		<rdfs:range rdf:resource="&fibo-fnd-org-fm;Employer"/>
+		<owl:inverseOf rdf:resource="&fibo-fnd-org-fm;isEmployingParty"/>
+		<skos:definition>identifies employer in an employment situation</skos:definition>
+	</owl:ObjectProperty>
+	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-org-fm;isDomiciledIn">
 		<rdfs:label>is domiciled in</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fnd-org-fm;FormalOrganization"/>
 		<rdfs:range rdf:resource="&lcc-cr;GeopoliticalEntity"/>
 		<skos:definition>indicates a primary location where an entity conducts business, such as where its headquarters is located</skos:definition>
```

Comment:
by definition surely you can only have one primary location, so "primary" seems wrong.

Reply:
fixed